### PR TITLE
docker db-sync

### PR DESCRIPTION
**Pull request by redoracle**
docker-compose db-sync

### CODE REVIEW

File: files/docker/db-sync/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      - node-ipc:/opt/cardano/cnode/sockets
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "10"
+
+  cardano-db-sync:
+    image: inputoutput/cardano-db-sync:alonzo-white-1.4
```
Comment (rdlrt) [inline files/docker/db-sync/docker-compose.yml:49]: did you want to add purple here?

File: files/docker/db-sync/guildsetup/cfg/postgresql.conf [hunk 2]
```diff
@@ -0,0 +1,705 @@
...
+#ssl_passphrase_command_supports_reload = off
+
+
+#------------------------------------------------------------------------------
+# RESOURCE USAGE (except WAL)
+#------------------------------------------------------------------------------
+
+# - Memory -
+
+shared_buffers = 128MB			# min 128kB
```
Comment (rdlrt) [inline files/docker/db-sync/guildsetup/cfg/postgresql.conf:120]: We've found this value to be extremely low/ill-effective when IOPs gets lower

**Review by rdlrt — APPROVED** [review]

File: files/docker/db-sync/guildsetup/cfg/postgresql.conf (hunk 2 above)
Comment (redoracle) [reply files/docker/db-sync/guildsetup/cfg/postgresql.conf:120]: yes that's still a demo file not yet actively used

File: files/docker/db-sync/docker-compose.yml (hunk 1 above)
Comment (redoracle) [reply files/docker/db-sync/docker-compose.yml:49]: sure

**Pull request merged**